src/site/content/en/blog/read-files/index.md

```diff
@@ -261,4 +270,4 @@ Browser compatibility data for the APIs used in this section:
 [text]: https://developer.mozilla.org/docs/Web/API/FileReader/readAsText
 [dropeffect]: https://developer.mozilla.org/docs/Web/API/DataTransfer/dropEffect
 [propagation]: https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Examples#Example_5:_Event_Propagation
-[preventdefault]: https://developer.mozilla.org/docs/Web/API/Event/preventDefault
\ No newline at end of file
+[preventdefault]: https://developer.mozilla.org/docs/Web/API/Event/preventDefault
```

Comment:
I don't see what's different here. Was there supposed to be a change?

Reply:
My editor removed a trailing space.